util/src/main/java/org/hyperledger/besu/util/bytes/Bytes32.java

```diff
@@ -168,6 +169,10 @@ static Bytes32 fromHexStringStrict(final String str) {
     return wrap(BytesValues.fromRawHexString(str, -1, false));
   }
 
+  static Bytes32 fromPlugin(final UnformattedData data) {
+    return wrap(data.getByteArray());
```

Comment:
@shemnon Would we want to leave this one as is since wrap here does runtime verification of the structure?

Reply:
If it's a Bytes32 it should already have been validated when the instance was made.  If we keep wrapping this will put pressure on the garbage collector and I'd like to avoid that as much as possible.  Either way.

Reply:
ah right, sounds good